package.json

```diff
@@ -4,58 +4,60 @@
   "version": "12.4.0",
   "dependencies": {
     "bootstrap": "^3.4.1",
-    "classnames": "^2.2.6",
-    "core-js": "^3.2.1",
```

Comment:
ozaj, `core-js` uz netreba?

Reply:
No myslím že ne. Nikde v src není. Tam to asi bylo jak se řešilo IE někdy nebo tak.

Reply:
My tam máme preset-env a preset-react, kde obojí mají nastavené `useBuiltIns` default na false. V tom případně `core-js` není potřeba. Snad to chápu dobře.

- https://babeljs.io/docs/en/babel-preset-env#usebuiltins
- https://babeljs.io/docs/en/babel-preset-react#usebuiltins

Reply:
Jo, chapem to rovnako. Navyse to produkuje rovnaky vystup.